modules/serverless-build-func-kn.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// serverless/serverless-functions-quickstart.adoc
+// add user guide?
+
+[id="serverless-build-func-kn_{context}"]
+= Building functions
+
+Before you can run a function, you must build the function project by using the `kn func build` command. The build command reads the `func.yaml` file in the directory for your function to determine the image name and registry.
```

Comment:
About the second phrase, it seems a little confusing to me. As suggestion, what about:
 
The build command reads the `func.yaml` file from the function project directory to determine the image name and registry